services/movie.py

```diff
@@ -0,0 +1,32 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(genres_ids: list[int] = None,
+               actors_ids: list[int] = None
+               ) -> QuerySet:
+
+    movies_set = Movie.objects.all()
+    if genres_ids:
+        movies_set = movies_set.filter(genres__id__in=genres_ids)
+    if actors_ids:
+        movies_set = movies_set.filter(actors__id__in=actors_ids)
+    return movies_set
+
+
+def get_movie_by_id(movie_id: int) -> QuerySet:
+    return Movie.objects.get(id=movie_id)
+
+
+def create_movie(movie_title: str, movie_description: str,
+                 genres_ids: list[int] = None, actors_ids: list[int] = None
+                 ) -> None:
```

Comment:
use typehint like: Optional[YOUR_VALUE_TYPE] = None
for example: list that contain string elements and can be None has typehint Optional[list[str]]

format it nicer